lib/cli/interactive-setup/aws-credentials.js

```diff
@@ -3,15 +3,72 @@
 const chalk = require('chalk');
 const _ = require('lodash');
 const inquirer = require('@serverless/utils/inquirer');
+const memoizee = require('memoizee');
 
 const AWS = require('aws-sdk');
 
 const awsCredentials = require('../../plugins/aws/utils/credentials');
 const { confirm } = require('./utils');
 const openBrowser = require('../../utils/openBrowser');
+const ServerlessError = require('../../serverless-error');
 
 const isValidAwsAccessKeyId = RegExp.prototype.test.bind(/^[A-Z0-9]{10,}$/);
 const isValidAwsSecretAccessKey = RegExp.prototype.test.bind(/^[a-zA-Z0-9/+]{10,}$/);
+const { getPlatformClientWithAccessKey } = require('@serverless/dashboard-plugin/lib/clientUtils');
+
+const CREDENTIALS_SETUP_CHOICE = {
+  LOCAL: '_local',
+  CREATE_PROVIDER: '_create_provider_',
+  SKIP: '_skip_',
+};
+
+const getProviderLinkUid = (app, service) => `appName|${app}|serviceName|${service}`;
+
+const getSdkInstance = memoizee(
+  async (orgName) => {
+    return getPlatformClientWithAccessKey(orgName);
+  },
+  { promise: true }
+);
+
+// TODO: TESTS
+const getOrgUidByName = memoizee(
```

Comment:
If we have use cases for same utils in context of `dashboard-plugin` it might be nice to seclude it there to to some utils folder, and reuse them in both locations

Reply:
good idea, I'll see if it makes sense to have it there